config/api.yml

```diff
@@ -1883,8 +1883,22 @@
     :options:
     - :collection
     - :subcollection
-    :verbs: *g
+    :verbs: *gp
     :klass: MiqRequestTask
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_request_show
+    :subcollection_actions:
+      :get:
+      - :name: read
+        :identifier: miq_request_show
+      :post:
+      - :name: edit
+        :identifier: miq_request_edit
+    :subresource_actions:
+      :post:
+      - :name: edit
```

Comment:
Does this mean that anyone can edit a single request task?

Reply:
@mkanoor I think then that this needs the `miq_request_edit` identifier as in 1898.